packages/button/src/docs/Component.stories.mdx

```diff
@@ -28,16 +28,6 @@ import variantStyles from '!!raw-loader!../styles/variant.module.css';
         disable: true,
       },
     },
-    leftIcon: {
-      table: {
-        disable: true,
-      },
-    },
-    rightIcon: {
-      table: {
-        disable: true,
-      },
-    },
```

Comment:
in the sandbox, it should stay hiding. because it is not really useful and no really convenient to use these options in the sandbox.